ni_measurementlink_service/measurement/service.py

```diff
@@ -7,9 +7,10 @@
 from os import path
 from pathlib import Path
 from threading import Lock
-from typing import Any, Callable, Dict, List, Optional, Type, TypeVar
+from typing import Any, Callable, Dict, List, Optional, Type, TypeVar, Union
 
 import grpc
+from google.protobuf.internal.enum_type_wrapper import EnumTypeWrapper
```

Comment:
This is internal, so I'm not sure we're allowed to reference it.

If we must import an internal type, we can reduce the risk of breakage by doing the import only when type checking, not at run time.

```
from __future__ import annotations
...
import typing
...

if typing.TYPE_CHECKING:
   from google.protobuf.internal.enum_type_wrapper import EnumTypeWrapper
```

Reply:
Let me know if I'm misinterpreting this situation, but only importing during type checking doesn't seem to work as we use `EnumTypeWrapper` for more than just `SupportedEnumType`. We reference `EnumTypeWrapper` as part of generating the annotations for the enum (in `MeasurementService._enum_to_annotations_value`) when we resolve the configuration and output decorators. I tried to find a public alternative to `EnumTypeWrapper` that we could use instead to effectively make the same check but couldn't find anything that panned out. I've left this as-is for now.

Reply:
Updated to import EnumTypeWrapper only on type checking and swapped the runtime `isinstance` usage via the EnumDescriptor instead. Thanks for the suggestion here.